package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ep-store-monorepo",
   "private": true,
-  "version": "3.0.9",
+  "version": "3.0.10",
```

Comment:
Thanks for updating this one @petro97! In the future we'll co-ordinate versions better. I'll post the release notes for 3.0.10 later today.